keps/sig-node/20190226-pod-overhead.md

```diff
@@ -134,117 +158,38 @@ Expand the runtimeClass type to include sandbox overhead, `Overhead *ResourceReq
  (runtimeClass selectors?) which would then be consumed by an admission controller and checked against known
  capabilities on a per runtimeClass basis. This is is beyond the scope of this proposal.
- -Automatic runtimeClass selection: A controller could exist which would attempt to automatically select the
+ - Automatic runtimeClass selection: A controller could exist which would attempt to automatically select the
  most appropriate runtimeClass for the given pod. This, again, is beyond the scope of this proposal.
 
-### User Stories [optional]
+### Implementation Details
 
-Detail the things that people will be able to do if this KEP is implemented.
-Include as much detail as possible so that people can understand the "how" of the system.
-The goal here is to make this feel real for users without getting bogged down.
-
-#### Story 2
-
-### Implementation Details/Notes/Constraints [optional]
-
-What are the caveats to the implementation?
-What are some important details that didn't come across above.
-Go in to as much detail as necessary here.
-This might be a good place to talk about core concepts and how they releate.
+With this proposal, the following changes are required:
+ - Add the new API to the pod spec and RuntimeClass
+ - Update the RuntimeClass controller to merge the overhead into the pod spec
+ - Update the ResourceQuota controller to account for overhead
+ - Update the scheduler to account for overhead
+ - Update the kubelet (admission, eviction, cgroup limits) to handle overhead
 
 ### Risks and Mitigations
 
-What are the risks of this proposal and how do we mitigate.
-Think broadly.
-For example, consider both security and how this will impact the larger kubernetes ecosystem.
-
-How will security be reviewed and by whom?
-How will UX be reviewed and by whom?
-
-Consider including folks that also work outside the SIG or subproject.
+This proposal introduces changes across several Kubernetes components and a change in behavior *if* Overhead fields
+are utilized. To help mitigate this risk, I propose that this be treated as a new feature with an independent feature gate.
 
 ## Design Details
 
-### Test Plan
-
-**Note:** *Section not required until targeted at a release.*
-
-Consider the following in developing a test plan for this enhancement:
-- Will there be e2e and integration tests, in addition to unit tests?
-- How will it be tested in isolation vs with other components?
-
-No need to outline all of the test cases, just the general strategy.
-Anything that would count as tricky in the implementation and anything particularly challenging to test should be called out.
-
-All code is expected to have adequate tests (eventually with coverage expectations).
-Please adhere to the [Kubernetes testing guidelines][testing-guidelines] when drafting this test plan.
-
-[testing-guidelines]: https://git.k8s.io/community/contributors/devel/sig-testing/testing.md
-
-### Graduation Criteria
```

Comment:
nit: I would leave this section here, and just say "TBD". Having graduation criteria explicitly stated is a prerequisite for beta.

Reply:
this is a new feature gate, we should probably name the feature gate, and we can say criteria to be defined before reaching beta

Reply:
done.